<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>K-Converter</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-run">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="Tree">
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="actionTools">
   <property name="windowTitle">
    <string>Actions</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionConvert"/>
   <addaction name="actionReplace"/>
   <addaction name="actionSetvalue"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="optionTools">
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStop"/>
   <addaction name="actionEdit"/>
  </widget>
  <action name="actionConvert">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Edit sources</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset theme="edit-find-replace">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSetvalue">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>Set value</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionConvert</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ConvertActionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>EditActionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenActionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveActionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConvertActionClicked()</slot>
  <slot>EditActionClicked()</slot>
  <slot>OpenActionClicked()</slot>
  <slot>SaveActionClicked()</slot>
 </slots>
</ui>
